base/src/main/java/io/spine/reflect/Types.java

```diff
@@ -74,4 +76,32 @@ private Types() {
                                                  .getType();
```

Comment:
Something wrong with the formatting here and two lines above. Please adjust, as long as you are modifying the codebase right next to this spot.